.github/workflows/release.yml

```
@@ -1,7 +1,7 @@
 name: Release
 
 on:
-  workflows_run:
+  workflow_run:
     workflow: [Lint and Test]
```

```suggestion
workflows: [Lint and Test]
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#about-yaml-syntax-for-workflows